test/constants.spec.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-env mocha */
```

Comment:
I don't think we have to test this. Constants is basically a configuration file with no logic...